stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -25,9 +25,13 @@ public func _stdlib_mkstemps(_ template: inout String, _ suffixlen: CInt) -> CIn
   var utf8 = template.nulTerminatedUTF8
   let (fd, fileName) = utf8.withUnsafeMutableBufferPointer {
     (utf8) -> (CInt, String) in
-    let fd = mkstemps(UnsafeMutablePointer(utf8.baseAddress!), suffixlen)
-    let fileName = String(cString: utf8.baseAddress!)
-    return (fd, fileName)
+    return utf8.baseAddress!.withMemoryRebound(
+      to: CChar.self, capacity: Int(suffixlen)) {
```

Comment:
https://github.com/apple/swift/pull/3816 was merged, and the property `nulTerminatedUTF8` is gone.  Please switch to `utf8CString` and I think you won't need to rebind the memory then.